source/anti-patterns/language.md

```diff
@@ -100,7 +90,7 @@ In most cases, `$NIX_PATH` is set to the latest channel when Nix is installed, a
 
 :::{note}
 [Channels](https://nixos.wiki/wiki/Nix_channels) are a way of distributing Nix software.
-They are being phased out, but still used by default.
+They are being phased out by flakes, but still used by default.
```

Comment:
Let's not recommend flakes yet, but now we have this page giving some flakes context: https://nix.dev/concepts/flakes#flakes